services/importShop1.js

```diff
@@ -0,0 +1,109 @@
+const axios = require('axios')
+const cheerio = require('cheerio')
+
+const path = require('path')
+require('dotenv').config({ path: path.resolve(__dirname, '../.env') })
+const { SHOP1_NAME, SHOP1_URL, SHOP1_BASE_URL } = process.env
+
+const parsedLinks = []
+const pageLimit = 80
+let pageCounter = 1
+
+console.log(`\n  Scraping of ${SHOP1_URL} initiated...\n`)
+
+const getProductLinks = async url => {
+  try {
+    const response = await axios.get(url)
+    const $ = cheerio.load(response.data)
+
+    $('.row > div > .productListing > li')
+      .not('.soldout')
+      .map((i, el) => {
+        const link = SHOP1_BASE_URL + $(el).find('.pQuickView > a').attr('href')
+
+        const metadata = {
+          link: link,
+        }
+        parsedLinks.push(metadata)
+      })
+    const nextPageLink =
+      SHOP1_URL + `/page/${pageCounter}/sort/relevance/perpage/36`
+    console.log(`  Scraping: ${nextPageLink}`)
```

Comment:
Blank spaces intended within `console.log`?

Reply:
only for formatting the output